src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -130,11 +130,19 @@ public Integer updatePosition(@GraphQLRootContext Map<String, Object> context,
       throw new WebApplicationException("Couldn't process position update", Status.NOT_FOUND);
     }
 
+    final Position current = dao.getByUuid(pos.getUuid());
```

Comment:
If you want this to reflect the original Position object from the database, you should really retrieve it _before_ the `dao.update()` on line 128…
I would also suggest to name the object `existing` instead of `current` to be in line with other places where this idiom is used.